CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Changelog
 
+- Fixes and improvements for SpecApi [#5198](https://github.com/MakieOrg/Makie.jl/pull/5198).
```

Comment:
maybe say what was fixed? otherwise it's not so useful for the changelog I think